### ISSUE TITLE
one feedback about your module

### ISSUE THREAD

**Issue by marcusau** [jidasheng/bi-lstm-crf#4]
Very good and user friendly.

I train my model on a non-GPU PC and the training time of 660000 sentences corpus is less than 30 mins.

Very amazing.

My only concern is about accuracy because 20 epochs is not enough for my corpus ,maybe due to huge size.

The loss , val_loss are all about 50. sth.

I am now increasing the epochs from 20 to 100 and see the result.

Thanks for your nice work

**Comment by jidasheng** [comment]
you can draw the learning curve and see whether it had converged.
